tests/gcp/installer/util/test_data/merged_bucket_rules.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
```

Comment:
IIUC, these `merged_` should be better renamed as `expected_merged_bucket_rules.yaml` file.